### PR TITLE
[biswap] Add biswap strategy

### PR DESCRIPTION
**Pull request by biswap-dex**
- Balances: auto compound, pool BSW - BSW, farm BSW - BNB and BSW USDT, launchpools, token

### CODE REVIEW

File: src/strategies/biswap/index.ts [hunk 1]
```diff
@@ -0,0 +1,276 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits, parseUnits } from '@ethersproject/units';
+import Multicaller from '../../utils/multicaller';
+import examplesFile from './examples.json';
+
+export const author = 'biswap';
```
Comment (ChaituVR) [inline src/strategies/biswap/index.ts:6]:
```suggestion
export const author = 'biswap-dex';
```

File: src/strategies/biswap/index.ts [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+      {
+        "name": "balance",
+        "type": "uint256"
+      }
+    ],
+    "payable": false,
+    "stateMutability": "view",
+    "type": "function"
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/biswap/index.ts:78]:
```suggestion
const masterChefAbi = [
  'function userInfo(uint256, address) view returns (uint256 amount, uint256 rewardDebt)',
  'function totalSupply() view returns (uint256)',
  'function balanceOf(address _owner) view returns (uint256 balance)'
];
```

File: src/strategies/biswap/index.ts [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+      {
+        "internalType": "uint256",
+        "name": "lastUserActionTime",
+        "type": "uint256"
+      }
+    ],
+    "stateMutability": "view",
+    "type": "function"
+  },
+];
```
Comment (ChaituVR) [inline src/strategies/biswap/index.ts:155]:
```suggestion
const autoBswAbi = [
  'function userInfo(address) view returns (uint256 amount, uint256 rewardDebt)'
];
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by ChaituVR — COMMENTED** [review]
LGTM

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
@biswap-dex Strategy is live now: https://snapshot.org/#/strategy/biswap

**Pull request merged**